docs/hpc/09_ood/r_packages_with_renv.md

```diff
@@ -0,0 +1,197 @@
+# R Packages with renv
```

Comment:
Move this to `06_tools_and_software` since it fits better there?

Reply:
Sounds good. will do.